README.md

```diff
@@ -12,109 +12,831 @@ The repository is a collection of open-source implementation of a variety of alg
 Read our [Contribution Guidelines](CONTRIBUTING.md) before you contribute.
 
 ## List of Algorithms
+<!--- GODOCMD BEGIN --->
```

Comment:
NOTE: This is necessary for my godocmd action to work otherwise it will append to the file.